db/dcrsqlite/apisource.go

```diff
@@ -1501,3 +1501,13 @@ func (db *wiredDB) TxHeight(txid string) (height int64) {
 	height = txraw.BlockHeight
 	return
 }
+
+//GetDifficulty returns the difficulty
```

Comment:
Two notes:
1. We are trying to avoid functions starting with `Get`, so just `Difficulty` would be good.
2. Comment formatting needs to be consistent, with a space between `//` and the function name. The docs should be full sentences ending in a period.